src/components/MapComponent.vue

```diff
@@ -0,0 +1,335 @@
+<template>
+  <div id="map-component">
```

Comment:
🌱 How much effort would it take to have a satellite view on the map? I feel like that's much more fitting.